### PR TITLE
feat(replays): Create replay specific endpoint for fetching errors with check_global_views=False

### LINKED ISSUE #38812: Fetching Errors fails in my test org, but not on emerging-tech or sentry org
**Problem:** 

My request
Request URL: https://dev.getsentry.net:7999/api/0/organizations/wildcard-inc/events/?end=2022-09-14T04%3A52%3A30.000Z&field=id&field=error.value&field=timestamp&field=error.type&field=issue.id&projects=-1&query=id%3A%5B990ed9f1ed6b4f96938fe22a7032b21c%5D&referrer=api.replay.details-page&start=2022-09-14T04%3A51%3A20.000Z
is returning 400 {"detail":"You cannot view events from multiple projects."} on a test org that i recently setup, but not when i run that request from the sentry org

**Issue:**
The test org does not have `organizations:global-views` enabled. Therefore it can't make requests to the events endpoint because there is a guard to prevent `>1 projects` from being allowed.

**Solution:**
We're going to make a new endpoint to support the replay use-case. The tip is:
> you should be able to copy something like `OrganizationEventsMetaEndpoint` in src/sentry/api/endpoints/organization_events_meta.py to get what you need
- via wmak 

existing call to update is here: https://github.com/getsentry/sentry/blob/c50743c63ef2da8bf175c9acc66ddcbd52bb0856/static/app/utils/replays/hooks/useReplayData.tsx#L173-L183

### PR DESCRIPTION
**Pull request by ryan953**
Fixes #38812

### CODE REVIEW

File: src/sentry/replays/endpoints/organization_replay_events_meta.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import NoProjects, OrganizationEventsV2EndpointBase
+from sentry.api.paginator import GenericOffsetPaginator
+from sentry.snuba import discover
+
+
+@region_silo_endpoint
+class OrganizationReplayEventsMetaEndpoint(OrganizationEventsV2EndpointBase):
```
Comment (JoshFerge) [inline src/sentry/replays/endpoints/organization_replay_events_meta.py:11]: needs `private = True` on the endpoint class

File: src/sentry/api/urls.py [hunk 2]
```diff
@@ -1563,6 +1566,11 @@
                     OrganizationReplayIndexEndpoint.as_view(),
                     name="sentry-api-0-organization-replay-index",
                 ),
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/replays_events_meta/$",
```
[the remark this answers is not in the document]
Comment (ryan953) [reply src/sentry/api/urls.py:1570]: kebab!

[the remark this answers is not in the document]
Comment (markstory) [reply src/sentry/api/urls.py]: Love kebabs ❤️

**Comment by ryan953** [comment]
@JoshFerge @markstory LMK if there are other changes/additions or if we can go ahead and merge

File: src/sentry/replays/endpoints/organization_replay_events_meta.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
```
Comment (JoshFerge) [inline src/sentry/replays/endpoints/organization_replay_events_meta.py:11]: could we add a comment here saying what the purpose of this endpoint is?

**Review by JoshFerge — APPROVED** [review]

**Review by JoshFerge — COMMENTED** [review]
could also write some endpoint tests to verify response schema, etc.

**Review by JoshFerge — APPROVED** [review]

**Pull request merged**